### ISSUE TITLE
Server crash 1.16.5

### ISSUE THREAD

**Issue by MagicFun1241** [Wilyicaro/BetterFurnacesReforged#39]
Java version: 11
Minecraft version: 1.16.5
Mod version: 1.9.1

```
java.lang.NoSuchFieldError: RECIPE_TYPE
        at wily.betterfurnaces.init.Registration.recipeRegister(Registration.java:56) ~[betterfurnacesreforged:1.16.5-1.9.1] {re:classloading}
        at wily.betterfurnaces.init.Registration.<clinit>(Registration.java:63) ~[betterfurnacesreforged:1.16.5-1.9.1] {re:classloading}
        at wily.betterfurnaces.BetterFurnacesReforged.<init>(BetterFurnacesReforged.java:53) ~[betterfurnacesreforged:1.16.5-1.9.1] {re:classloading}
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?] {re:classloading}
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?] {re:classloading}
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?] {re:classloading}
        at java.lang.reflect.Constructor.newInstance(Constructor.java:490) ~[?:?] {}
        at java.lang.Class.newInstance(Class.java:584) ~[?:?] {}
        at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[forge:36.2] {re:classloading}
        at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:120) ~[forge:?] {re:classloading}
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736) [?:?] {}
        at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1728) [?:?] {}
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290) [?:?] {}
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020) [?:?] {}
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656) [?:?] {}
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594) [?:?] {}
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183) [?:?] {}
```

**Comment by Wilyicaro** [comment]
Sorry.. I posted the incorrect 1.9.1 mod jar, I will repost this version

**Comment by Wilyicaro** [comment]
Fixed in version 1.18.2-1.2.3f

**Comment by Wilyicaro** [comment]
Fixed in version 1.18.2-1.2.3f
